docs/guide/overview.rst

```diff
@@ -0,0 +1,83 @@
+GUFE Overview
+=============
+
+GUFE exists to define interoperable APIs for free energy calculations, which
+can be used by an ecosystem of Python packages to develop tools that focus
+on performing specific aspects of the free energy pipeline, while
+benefitting from other tools for other aspects.
+
+In order to do this, GUFE distinguishes several aspects of the process of
+calculating free energies, and define APIs for each of them. GUFE provides
+the underlying infrastructure; the real science is done in packages that
+implement parts of the GUFE APIs.
+
+Core data models
+----------------
+
+In order to ensure interoperability, GUFE defines objects that represent the
+core chemistry and alchemistry of a free energy pipeline, including
+molecules, chemical systems, and alchemical transformations. This provides a
+shared language that different tools use.
+
```

Comment:
Is it worth having a brief section on `Components` here?

Reply:
I think the next page in the guide is going to be a discussion of the `Component`s approach (we already have a placeholder for it). My goal for this page is to be an extended table of contents for the rest of the "guide" section; details will come later.